charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```diff
@@ -53,6 +53,16 @@ spec:
             args:
               - {{ .Values.esIndexCleaner.numberOfDays | quote }}
               - {{ include "elasticsearch.client.url" . }}
+              {{- range $key, $value := .Values.esIndexCleaner.cmdlineParams -}}
+              {{- if $value }}
+              - --{{ $key }}={{ $value }}
+              {{- else }}
+              - --{{ $key }}
+              {{- end }}
+              {{- end -}}
+              {{- if not .Values.ingester.enabled -}}
```

Comment:
so if ingester is not enabled, we would add storage.cmdArgs otherwise not ? (can you elaborate on this a little please)

Reply:
It's how it is done on the collector and the query deployment, 
I thought maybe that had been done this way, so those args are only for the ingester if he's active. 
I didn't want to break that logic. :) I suppose the cronJobs can bypass this logic, you tell me? :)